components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -138,8 +138,31 @@ __attribute__((objc_subclassing_restricted))
  */
 @property(nonatomic, nullable, strong) UIFont *actionFont;
 
+/**
+ The color applied to the sheet view of the action sheet controller.
+ */
 @property(nonatomic, nonnull, strong) UIColor *backgroundColor;
 
+/**
+ The color applied to the title of Alert Controller.
+ */
+@property(nonatomic, strong, nullable) UIColor *titleColor;
+
+/**
+ The color applied to the message of Alert Controller.
+ */
+@property(nonatomic, strong, nullable) UIColor *messageColor;
```

Comment:
As discussed in https://github.com/material-components/material-components-ios/pull/4968#discussion_r214355546, I'm a bit worried that this API may not let us build a themer that guarantees a correct theme in cases where properties are set after theming. Is there another way we can structure this API to better support the themer?

Reply:
Think I found a solution.